claripy/frontend_mixins/smtlib_script_dumper_mixin.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import sys
+import threading
+
+from ..frontends.constrained_frontend import ConstrainedFrontend
+
+l = logging.getLogger(__name__)
+
+class SMTLibScriptDumperMixin(object):
```

Comment:
`(object)` is not necessary in Python 3.

Reply:
Fixed now